transactions/idTableStaking/get_node_info.cdc

```diff
@@ -0,0 +1,7 @@
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+
+// This script gets all the info about a node and returns it
+
+pub fun main(nodeID: String): FlowIDTableStaking.NodeInfo {
+    return FlowIDTableStaking.NodeInfo(nodeID: nodeID)
+}
```

Comment:
This is getting repetitive after a couple of first scripts 😅 
How about we do it this way? 🤔

```suggestion
pub fun main(nodeID: String, _ key: String?): AnyStruct {
    let nodeInfo = FlowIDTableStaking.NodeInfo(nodeID: nodeID)
    
    if key == "initialWeight" { return nodeInfo.initialWeight } 
    if key == "networkingAddress" { return nodeInfo.networkingAddress } 
    if key == "networkingKey" { return nodeInfo.networkingKey } 
    if key == "tokensRewarded" { return nodeInfo.tokensRewarded } 
    if key == "role" { return nodeInfo.role } 
    if key == "tokensStaked" { return nodeInfo.tokensStaked } 
    if key == "stakingKey" { return nodeInfo.stakingKey } 
    if key == "tokensCommitted" { return nodeInfo.tokensCommitted }
    if key == "tokensUnlocked" { return nodeInfo.tokensUnlocked } 
    if key == "tokensUnstaked" { return nodeInfo.tokensUnstaked } 
    if key == "tokensRequestedToUnstake" { return nodeInfo.tokensRequestedToUnstake } 
    
    // In other cases, return full nodeInfo
    return nodeInfo
}
```

Reply:
I don't know, I think it would probably be clearer to app developers if they were individual scripts for each field instead of one script for all of them. What do you think, @JeffreyDoyle @psiemens